src/plugins/cloudMetadata.nim

```diff
@@ -200,13 +200,18 @@ proc cloudMetadataGetrunTimeHostInfo*(self: Plugin, objs: seq[ChalkObj]):
   if isGoogleHost(vendor, resolv) and (
     isSubscribedKey("_GCP_INSTANCE_METADATA") or
     isSubscribedKey("_GCP_PROJECT_METADATA") or
+    isSubscribedKey("_OP_CLOUD_PROVIDER") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_IP") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_REGION") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_TAGS") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_ACCOUNT_INFO") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_SERVICE_TYPE") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_INSTANCE_TYPE")
```

Comment:
lets say only `_OP_CLOUD_PROVIDER` is subscribed but with this condition it will hitting metadata endpoint even if no other key is subscribed

I think it should be a separate conditional block outside this block which sets just the `_CLOUD_PROVIDER`. it will then keep the correct subscription semantics

same below